starter-code/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "starter-code",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev":"nodemon ./app.js -e .js,.hbs"
```

Comment:
`nodemon ./app.js -e .js,.hbs`
Can be 
 `nodemon ./app.js -e js,hbs`
According to the doc, though adding the . might work, never tried.
https://www.npmjs.com/package/nodemon/v/1.19.2#specifying-extension-watch-list